Modules/Filtering/BiasCorrection/include/itkN4BiasFieldCorrectionImageFilter.h

```diff
@@ -405,14 +405,14 @@ class N4BiasFieldCorrectionImageFilter :
   // Parameters for deconvolution with Wiener filter
 
   unsigned int m_NumberOfHistogramBins{ 200 };
-  RealType     m_WienerFilterNoise{ 0.01 };
-  RealType     m_BiasFieldFullWidthAtHalfMaximum{ 0.15 };
+  RealType     m_WienerFilterNoise{ static_cast< RealType >( 0.01 ) };
```

Comment:
{ 0.01F } would have also worked.  Then if someone tried to instantiate an ImageType::ScalarType = integer, the narrowing failure would alert them.  In this case, 0.01 will be narrowed to 0. 

I'm fine with your solution, but I do prefer to keep the { value } initializers to be explicit on the desired behavior.

Thanks.

Reply:
Ignore my comment here.  I think the static_cast<>() solution is the best in  this situation.